docs/terminal/brisbane.md

```diff
@@ -43,6 +44,9 @@ Aircraft shall be assigned the following runways for arrival when PROPS are in p
 | From the NORTH and WEST | 01L/19R |
 | From the SOUTH and EAST | 01R/19L |
 
+### Independent Visual Approach Phraseology at Night
+*"CLEARED INDEPENDENT VISUAL APPROACH RUNWAY (number), NOT BELOW (altitude) UNTIL ESTABLISHED ON THE VASIS (or PAPI) (or GLIDEPATH)"*
```

Comment:
Thoughts on replacing 'established on the VASIS (or PAPI) (or GLIDEPATH)' with just 'established on the PAPI or glidepath', since we don't do IVAs at any airports with VASIS?

Reply:
I just copied verbatim from MATS tbh

Reply:
Yeah I figured as much. Just wondering if we adjust it to make it more relevant for our use

Reply:
Yep fair